src/Uno.UWP/UI/Core/CoreWindow.Pointers.Managed.cs

```diff
@@ -24,6 +24,23 @@ public partial class CoreWindow : ICoreWindowEvents
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyDown;
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyUp;
 
+#if __SKIA__
```

Comment:
All code here might be best in another partial.

Reply:
After removing WrappedKeyEventArgs, I think it's small enough to not warrant a new partial?

Reply:
@ramezgerges My concern is that this file is more about "pointers"